lib/formatter/csv/v2/annotation.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copy of the AnnotationForCsv class bringing over the default annotation formatting behaviour
+# an updating to handel the "classifier_version"=>"2.0" as noted on
+# classification metadata FEM classifications vs PFE (v1 and missing on the metadata)
+#
+# Overtime this class will evolve behaviour to handle the distinct classification annotation formats
+# if it doesn't these two behaviours can be refactored for collaborating classes
+# to provide csv converstion of the differing annotation formats
+module Formatter
+  module Csv
+    module V2
+      class Annotation
+        attr_reader :classification, :current_annotation, :cache, :default_formatter, :workflow_information, :task_info
+
+        def initialize(classification, annotation, cache, default_formatter=nil)
```

Comment:
Is allowing `default_formatter` as an arg here for future use? Or for some other purpose? I get that having it in the v2 annotator lets you use the rest of the unchanged task formatters, but it's not being passed explicitly to the v2 annotator anywhere that I can find.

Reply:
dependency injection to allow us to easily change the fallback formatter without changing this class interface. It helps in testing as well (i.e. we can inject the double at the build time) - though i didn't do that as i let it build it's default. 

I'm going to leave this in for the extension point for class behaviour